Cargo.toml

```diff
@@ -16,8 +16,9 @@ package.repository = "https://github.com/matrix-org/matrix-authentication-servic
 unsafe_code = "forbid"
 
 [workspace.lints.clippy]
-all = "deny"
-pedantic = "warn"
```

Comment:
Cargo started complaining about this, because both a single lint (`str_to_string`) and group lints had the same priority